### ISSUE TITLE
Starter kit should include source code

### ISSUE THREAD

**Issue by felixrabe** [Famous/famous#286]
This is what happened: I was on an airport before a nine hour flight. I was busy getting some offline entertainment (i.e. Famous docs _and source code_) in my hands, so I downloaded the starter kit... Found out, there is no source code included in the starter kit. I finally managed to save `require.js` and `famous.min.js` from some open Chrome browser tabs and rewrite the boilerplate to point to the locally-downloaded files. No luck for the CSS though, but it was enough to fiddle with some examples.

**tl;dr** - please include the source code in the ZIP behind the big fat red "Download" button.

Btw, Famo.us is the first open source project that I encountered that did not provide any kind of executable code in a "Download" package.
